test/parallel/test-module-readonly.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+const cp = require('child_process');
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+if (common.isWindows) {
```

Comment:
Maybe it would be more optimal to add this check at the beginning, something like:
```js
const common = require('../common');

// TODO: Similar checks on *nix-like systems (e.g using chmod or the like)
if (!common.isWindows)
  common.skip('test only runs on Windows');

const assert = require('assert');
// ...
```

This way the test will not waste the time and recourses loading all the other modules. `common.skip()` makes the script exit, so no `else` is required after it and we can save one indentation level as well.